12_2CH_EMG_Game_Controller/12_2CH_EMG_Game_Controller.py

```
@@ -0,0 +1,84 @@
+                # Handle keystroke 2 (right key press and release after 100ms)
+                elif most_common == "2":
+                    if most_common not in keystroke_hold_time or current_time - keystroke_hold_time[most_common] >= 100:
+                        # Press the right key
+                        keyboard.press('right')
+                        print("right pressed")
+                        keystroke_hold_time[most_common] = current_time
+                        key_release_time[most_common] = current_time + 100  # Release after 100ms
+                
+                # Handle keystroke 3 (up key press and release after 100ms)
+                elif most_common == "3":
+                    if most_common not in keystroke_hold_time or current_time - keystroke_hold_time[most_common] >= 100:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)